Update npm peer dependencies to work with ESLint 2.0+

Hi,

I tried upgrading one of my projects to ESLint 2.0, but the upgrade failed due to peer dependency conflicts with eslint-config-semistandard

> npm ERR! peer dep missing: eslint@^1.0.0, required by eslint-config-semistandard@5.0.0
npm ERR! peer dep missing: eslint@^1.6.0, required by eslint-loader@1.2.1
npm ERR! peer dep missing: eslint-config-standard@^4.0.0, required by eslint-config-semistandard@5.0.0
npm ERR! peer dep missing: eslint-plugin-promise@^1.0.8, required by eslint-config-standard@5.1.0

Can you please update your dependencies to work with ESLint 2.0+?

Thanks!

--- a/package.json
+++ b/package.json
@@ -7,10 +7,10 @@
     "url": "https://github.com/Flet/eslint-config-semistandard/issues"
   },
   "devDependencies": {
-    "eslint": "^1.0.0",
-    "eslint-config-standard": "^4.0.0",
-    "eslint-plugin-standard": "^1.2.0",
-    "standard": "^5.0.0",
+    "eslint": "^2.0.0",
+    "eslint-config-standard": "^5.0.0",
+    "eslint-plugin-standard": "^1.3.0",
+    "standard": "^6.0.0",
     "tap-spec": "^4.0.0",
     "tape": "^4.0.0"
   },
@@ -52,6 +52,6 @@
     "test": "standard && tape test/*.js | tap-spec"
   },
   "peerDependencies": {
-    "eslint-config-standard": "^4.0.0"
+    "eslint-config-standard": "^5.0.0"
   }
 }

